src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -68,6 +93,11 @@ function ExportAnnotations({
       allAnnotationsOption,
   );
 
+  const exportFormatsEnabled = store.isFeatureEnabled('export_formats');
+  const [selectedExportFormat, setSelectedExportFormat] = useState(
```

Comment:
`selectedExportFormat` feels a bit verbose to me. I think `exportFormat/`setExportFormat` would be fine.